bindings/wallet-cordova/scripts/run_tests.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python
+
+import subprocess
+
+from pathlib import Path
+import subprocess
+import sys
+import shutil
+import argparse
+import os
+import re
+
+from build_jni import run as build_jni
+from build_jni import copy_libs as copy_jni_libs
+from copy_jni_definitions import run as copy_jni_definitions
+from build_ios import run as build_ios
+from directories import repository_directory, plugin_directory, tests_directory
+
+
+def sed(original: str, replacement: str, file: Path):
+    # TODO: this may have some problems, but I'm also not sure if I want to use
+    # `sed`, mostly for Windows compatibility
+    with open(file, "r") as config:
+        lines = config.readlines()
+
+    with open(file, "w") as config:
+        for line in lines:
+            config.write(re.sub(original, replacement, line))
+
+
+def create_hello_world(build_dir: Path):
+    os.makedirs(build_dir, exist_ok=True)
+
+    subprocess.check_call(
+        ["cordova", "create", "hello", "com.example.hello", "HelloWorld"],
+        cwd=build_dir,
+    )
+
+
+def install_test_framework(app_dir: Path):
+    subprocess.check_call(
+        ["cordova", "plugin", "add", "cordova-plugin-test-framework"], cwd=app_dir
+    )
+
+    sed(
+        '<content src="index.html" />',
+        '<content src="cdvtests/index.html" />',
+        app_dir / "config.xml",
+    )
+
+
+def install_platforms(app_dir: Path, android=True, ios=True):
+    if android:
+        subprocess.check_call(["cordova", "platform", "add", "android"], cwd=app_dir)
+        subprocess.check_call(["cordova", "requirements", "android"], cwd=app_dir)
+
+    if ios:
+        subprocess.check_call(["cordova", "platform", "add", "ios"], cwd=app_dir)
+        subprocess.check_call(["cordova", "requirements", "ios"], cwd=app_dir)
```

Comment:
Do we really need that call? Worked fine me without doing that for iOS.

Reply:
not *really* if you have things set up already... It doesn't have side-effects, it only validates that you have things installed.

I used it to find out what was missing from the ci, but after I fixed that it doesn't really have much purpose. I left it because it didn't seem like it was particularly slow, but we can remove it if it is.